src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -297,6 +290,21 @@
            type (some :type [props @state])
            id (some :id [props @state])]
        [:div {:key (str id)}
+        (when (:editing-namespace-acl? @state)
+          (let [{:keys [edit-namespace edit-type]} @state]
+            [mca/AgoraPermsEditor
+             {:dismiss #(swap! state dissoc :editing-namespace-acl?)
+              :save-endpoint (endpoints/post-agora-namespace-acl edit-namespace (= :config edit-type))
+              :load-endpoint (endpoints/get-agora-namespace-acl edit-namespace (= :config edit-type))
+              :entityType "Namespace" :entityName edit-namespace
+              :title (str "Namespace " edit-namespace)}]))
+        (when (:creating? @state)
+          [create/CreateMethodDialog
```

Comment:
I'm confused as to why these components are used here and as modals from the sidebar component.